server/monitor-types/websocket.js

```diff
@@ -0,0 +1,75 @@
+const { MonitorType } = require("./monitor-type");
+const WebSocket = require("ws");
+const { UP, DOWN } = require("../../src/util");
+const childProcessAsync = require("promisify-child-process");
+
+class websocket extends MonitorType {
```

Comment:
Please capitalise classes like all other classes in the project.
Weird that CI did not catch this.

Please also change the name in other usages

```suggestion
class WebSocketMonitorType extends MonitorType {
```

Reply:
Noted and fixed.